### ISSUE TITLE
Check for standardized xref prefixes in ontologies

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBOFoundry.github.io#525]
Many ontologies use xrefs to connect to external IDs. In OBO we generally prefer logical axioms, but xrefs are frequently used (1) where axiomatization is not yet complete (2) the ID linked to has no standard semantic representation and we intend a non-logical axiom (3) to supplement a logical axioms, as some tools like OxO only use the xrefs.

We should try and standardize these as much as possible. Some tools like OxO may use identifiers.org synonyms to unify heterogeneous representations, but we would like to avoid forcing xref consuming tools to do this. We should encourage use of standard registries

 * [GO xrefs](https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml)
 * identifiers.org
 * prefixcommons.org

We also have a quandary as to whether to use literals containing CURIEs or URIs. For historic reasons going back to oboedit most ontologies use the former. Also frequently there is no canonical URI for a non-OBO ID. Tools such as OxO assume this. Protege now supports CURIEs and will make nice clickable links (using id.org as a lookup I believe).

However, we should also make things  semweb friendly

I propose that as part of a release pipeline, we include both oio:xref and more explicit axioms to IRIs, either logical or non-logical.

An example is uberon which provides both xrefs to anatomy ontologies, and logical axioms
http://uberon.github.io/downloads.html#bridge

But currently we don't typically do this for non-logical xrefs. Here I propose that in addition to the oio:xref to a literal we use rdfs:seeAlso, isAbout, etc to a URI, using id.org or prefixcommons to auto-expand during release pipeline.

**Comment by nlharris** [comment]
Is this fixed by the PR above?

**Comment by matentzn** [comment]
This is far from done end very important. What it really needs is an automated check in some future advanced dashboard.

**Comment by amirlad** [comment]
We have a dataset of almost 100 OBO and OWL ontologies related to agronomy and biodiversity stored in our ontology repository [1]. We are currently having similar issues to identify the right target Ontologies and URIs in the Xrefs properties.  To solve this issue, we have developed a tool [2] that searches in identifiers.org, obo foundary repository and a local file (maintained semi-automatically) for the right target ontologies and URIs. Even after running our tool and spending a lot of time in searching the right Xrefs prefixes, we result in a lot of undiscovered prefixes. I see a need for standardization in here.  
Cc [jonquet](https://github.com/jonquet)
[1] http://agroportal.lirmm.fr/
[2] https://github.com/agroportal/ontology_mapping_harvester

**Comment by jonquet** [comment]
Hi all, we are preparing an ICBO submission with @amirlad on an analysis of the use of XRefs in 30 ontologies from the OBO world (some in the library others not). 
I will share a draft offline one submitted if interested. It will be very relevant for this issue.

**Comment by jonquet** [comment]
Related tracker is also here: 
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/826

**Comment by nlharris** [comment]
Is this still relevant and actionable?

**Comment by matentzn** [comment]
Yes there are a lot of debates on this across email, slack and OMO repo.. 

- the General problem: https://github.com/information-artifact-ontology/ontology-metadata/issues/59
- how to represent prefix map: https://github.com/information-artifact-ontology/ontology-metadata/issues/93

I would like to have an iterative approach with minimal changes to current practice, as outlined in the first ticket. But yeah this issue is at least underway.
